src/main/java/core/basesyntax/db/UserStorage.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.UserEmail;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class UserStorage {
+    public static final List<UserEmail> persons = new ArrayList<>();
```

Comment:
do we need this storage?
better remove it

Reply:
At first I make second storage, then decided to make another list in the same storage, but forgot to remove it...
fixed